python-lib/gluonts_forecasts/model.py

```diff
@@ -61,7 +62,8 @@ def __init__(
         season_length=None,
         mxnet_context=None,
     ):
-        super().__init__(model_name)
+        self.model_handler = ModelHandlerRegistry().get_model(model_name)
```

Comment:
IMO you could just call this `self.model`, it would be simpler.

Reply:
If we change the name of the ModelHandle class, it could be done